app/konnect/servicehub/index.md

```diff
@@ -1,91 +1,17 @@
 ---
-title: Service Hub Overview
+title: The Konnect Service Hub
+subtitle: Track every service accross your architecture
+content-type: explanation
 ---
 
-The Service Hub is a {{site.konnect_saas}} functionality module that
-lets you catalog all of your services in a single system of record. This
-catalog represents the single source of truth of your organization’s service
-inventory and their dependencies.
+The Service Hub makes internal APIs discoverable, consumable, and reusable for internal development teams. Catalog all your services through the Service Hub to create a single source of truth for your organization’s service inventory. By leveraging Service Hub, your application developers can search, discover, and consume existing services to accelerate their time-to-market, while enabling a more consistent end-user experience across the organization’s applications.
 
-Using the Service Hub, you can catalog, manage, and track every service in your
-entire architecture.
+![{{site.konnect_short_name}} service hub](/assets/images/docs/konnect/konnect-servicehub.png)
 
-## Services in the Service Hub catalog
 
-Each entry in the Service Hub is called a _{{site.konnect_short_name}} service_, or _service_.
-A {{site.konnect_short_name}} service is an abstraction of an upstream service.
+**[Access the {{site.konnect_short_name}} service hub](https://cloud.konghq.com/us/servicehub).**
```

Comment:
I thought this was a little buried, maybe put this before the screenshot with one of those arrows showing it's an external link?